rules-tests/DeadCode/Rector/If_/RemoveUnusedNonEmptyArrayBeforeForeachRector/Fixture/nullable.php.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace Rector\Tests\DeadCode\Rector\If_\RemoveUnusedNonEmptyArrayBeforeForeachRector\Fixture;
+
+function run(?array $values) {
+    if (!empty($values)) {
+        foreach ($values as $value) {
+            echo $value;
+        }
+    }
+}
+?>
+-----
+<?php
+namespace Rector\Tests\DeadCode\Rector\If_\RemoveUnusedNonEmptyArrayBeforeForeachRector\Fixture;
+
+function run(?array $values) {
+    if (!empty($values)) {
+        foreach ($values as $value) {
+            echo $value;
+        }
+    }
+}
+?>
```

Comment:
since the intent of this test is, that the code should not change, you can drop everything (and including) the `-----` line.

also: the fixture name should start with `skip_` (convention)